cypress/fixtures/teams2021.json

```diff
@@ -0,0 +1,36 @@
+{ "response":
+  [
+    {
+      "points": 600, 
+      "team": {"name": "Mercedes-AMG Petronas", "logo": "https://media-1.api-sports.io/formula-1/teams/5.png"}
+    },
+    {
+      "points": 500, 
+      "team": {"name": "Red Bull Racing", "logo": "https://media-3.api-sports.io/formula-1/teams/1.png"}
+    },
+    {
+      "points": 450, 
+      "team": {"name": "Example 1", "logo": "https://media-3.api-sports.io/formula-1/teams/1.png"}},
+    {
+      "points": 400, 
+      "team": {"name": "Example 2", "logo": "https://media-3.api-sports.io/formula-1/teams/1.png"}},
+    {
+      "points": 350, 
+      "team": {"name": "Example 3", "logo": "https://media-3.api-sports.io/formula-1/teams/1.png"}},
+    {
+      "points": 300, 
+      "team": {"name": "Example 4", "logo": "https://media-3.api-sports.io/formula-1/teams/1.png"}},
+    {
+      "points": 250, 
+      "team": {"name": "Example 5", "logo": "https://media-3.api-sports.io/formula-1/teams/1.png"}},
+    {
+      "points": 200, 
+      "team": {"name": "Example 6", "logo": "https://media-3.api-sports.io/formula-1/teams/1.png"}},
+    {
+      "points": 100, 
+      "team": {"name": "Example 7", "logo": "https://media-3.api-sports.io/formula-1/teams/1.png"}},
+    {
+      "points": 0, 
+      "team": {"name": "Example 8", "logo": "https://media-3.api-sports.io/formula-1/teams/1.png"}}
+  ]
+}
```

Comment:
I'm not sure if your algorithm requires 10 teams to run or not but it seems that the 8 example teams may be extraneous here..?

Reply:
I totally agree, I do have one conditional that depends on the length of the teams because there can only ever be 10 & if the user renames their team I have to splice out the 11th - it's a little weird and worthy of a refactor for sure.  I'll make sure to revisit this if I do have time for that refactor

Reply:
I figured that was probably the case! If your logic needs 10 teams to function then no need to refactor this IMO!